schema/consolidatedExtension/extension.json

```diff
@@ -1,4 +1,5 @@
 {
+  "REMOVE THIS TEST":"WHEN DONE",
```

Comment:
@timgdavies Is this intentional? cc @Bjwebb to check before deploy

Reply:
cc @duncandewhurst for time zone issues

Reply:
@bjwebb Apologies - yes - can confirm this line should come out.